### PR TITLE
Sessions tab

### PR DESCRIPTION
**Pull request by swemail**
Sessions tab with list of exercises.

- Reuses `CreateSessionModal`
- Extracts Fade and uses it for the modal
- Backend support for querying sessions by exerciseId

<image src="https://user-images.githubusercontent.com/1139207/220957323-0ed09739-b35c-417d-a47f-2e54aed4eca2.png" width="200" /> <image src="https://user-images.githubusercontent.com/1139207/220957642-2d688ae8-0e52-4410-8ea6-e22a313fe680.png" width="200" />

### CODE REVIEW

**Comment by gewfy** [comment]
FYI This one probably affects you #1941

**Comment by swemail** [comment]
> FYI This one probably affects you #1941

Indeed 😄

File: client/src/routes/screens/Sessions/Sessions.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+} from '../../../lib/navigation/constants/routes';
+import {formatExerciseName} from '../../../lib/utils/string';
+
+const BottomGradient = styled(LinearGradient)({
+  position: 'absolute',
+  left: 0,
+  right: 0,
+  bottom: 0,
+  height: 40,
+});
```
Comment (gewfy) [inline client/src/routes/screens/Sessions/Sessions.tsx:39]: I see that we've missed this in `Journey` tab too. Thinking about creating a reusable component or add it to the `Screen` component. But I can do that in a separate branch.

File: client/src/routes/screens/Sessions/Sessions.tsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+      <FlatList
+        data={exerciseIds}
+        renderItem={renderItem}
+        keyExtractor={keyExtractor}
+        ListHeaderComponent={
+          <Gutters>
+            <Heading16>{t('sessionsHeading')}</Heading16>
+            <Spacer8 />
+          </Gutters>
+        }
```
Comment (gewfy) [inline client/src/routes/screens/Sessions/Sessions.tsx:132]: Wouldn't it be good to make this sticky already now? And align it with `Home` and `Journey`.

**Review by gewfy — APPROVED** [review]

**Pull request merged**